torchtune/models/qwen2/_tokenizer.py

```diff
@@ -151,9 +151,7 @@ def __init__(
 
         self.pat = re.compile(PRETOKENIZE_REGEX)
 
-        self.special_tokens = (
-            special_tokens if special_tokens is not None else QWEN2_SPECIAL_TOKENS
-        )
+        self.special_tokens = special_tokens
```

Comment:
the reason this was none was because of some pythonic issue with using a dict as a default, it will make all instances of the Qwen2Tokenizer class share the same `QWEN2_SPECIAL_TOKENS`, and if a user were to ever modify it it woulud be changed for every single instance. don't know if that's an edge case we should really consider, but just fyi

Reply:
wouldn't that happen in the original way anyway? all instances initialized with special_tokens=none would share the same reference to the QWEN2_SPECIAL_TOKENS, right? i think this is fine since we don't modify it, but i could deepcopy it to be safe